### DISCUSSION TITLE
how to quit team repository?

### DISCUSSION THREAD

**Discussion by ghofrane28** [community/community#40733] [Repositories]
### Select Topic Area

Product Feedback

### Body

Hi
i want to lease a team repository but when i went to setting ->Code, planning, and automation-> repositories
i can't find the repository.
Any sugguestion please?

**Comment by mohamedzhioua** [comment]
@ghofrane28 If you no longer want to be a collaborator on someone else's repository, you can remove yourself just by following the steps in the link below from github documentation : "https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-access-to-your-personal-repositories/removing-yourself-from-a-collaborators-repository"
